File: PIPromptClases/MyApplication4/app/src/main/java/com/example/myapplication/Evento.kt
package com.example.myapplication

import java.time.LocalDateTime

data class Evento(
    var eventoId: Int,
    var nombre: String,
    var ubicacion: String,
    var descripcion: String,
    var imagen: String,
    var fecha: LocalDateTime
) {
    constructor() : this(
        eventoId = 0,
        nombre = "",
        ubicacion = "",
        descripcion = "",
        imagen = "",
        fecha = LocalDateTime.now()
    )
}

File: frontend-wpf/C#Clases/Evento.cs
public class Evento
    {
        public int EventoId { get; set; }
        public string Nombre { get; set; }
        public string Ubicacion { get; set; }
        public string Descripcion { get; set; }
        public string Imagen { get; set; }
        public DateTime Fecha { get; set; }

        public Evento() { }

        public Evento(int eventoId, string nombre, string ubicacion, string descripcion, string imagen, DateTime fecha)
        {
            EventoId = eventoId;
            Nombre = nombre;
            Ubicacion = ubicacion;
            Descripcion = descripcion;
            Imagen = imagen;
            Fecha = fecha;
        }
    }